frontend/public/locales/zh/machine-config-pools.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Resume updates": "恢复更新",
+  "Pause updates": "暂停更新",
+  "Max unavailable machines": "最大不可用机器",
+  "Current configuration": "当前配置",
+  "Current configuration source": "当前配置源",
+  "Paused": "暂停",
+  "True": "True",
+  "False": "False",
+  "Total machine count": "机器总数",
+  "Total number of machines in the machine pool.": "机器池中的机器总数。",
+  "machine_0": "机器",
+  "Ready machines": "就绪的机器",
+  "Total number of ready machines targeted by the pool.": "池目标就绪的机器总数。",
+  "Updated count": "更新的数量",
+  "Total number of machines targeted by the pool that have the CurrentMachineConfig as their config.": "池目标机器总数，将 CurrentMachineConfig 作为其配置。",
+  "Unavailable count": "不可用计数",
+  "Total number of unavailable (non-ready) machines targeted by the pool. A node is marked unavailable if it is in updating state or NodeReady condition is false.": "池目标的不可用（没有就绪）的机器总数。当节点处于更新状态或 NodeReady 条件为 false 时，节的会被标记为不可用。",
+  "MachineConfig selector": "MachineConfig 选择器",
+  "MachineConfigPool details": "MachineConfigPool 详情",
+  "Conditions": "条件",
+  "Name": "名称",
+  "Configuration": "Configuration",
+  "Updated": "已更新",
+  "Updating": "更新中",
+  "Degraded": "Degraded",
```

Comment:
"Configuration" and "Degraded" are not translated into Chinese.

Reply:
This has not been updated.

Reply:
I can see that this had been addressed in the translation files, will send the files over again just in case.

Reply:
Looks good to me; thanks!